### ISSUE TITLE
Fix Text that are designed as titles but not marked as headers via markup.

### ISSUE THREAD

**Issue by jeromehardaway** [Vets-Who-Code/vets-who-code-app#347]
"Titles built as text tags should be labeled as headings for assistive technology"

As a user, I should be able to know the importance of what I am on the web using by voiceover and HTML Markup tags. TO fix please identify the text that looks like titles but do not have proper HTML Tags and fix.

<p class="subtitle">With Vets Who Code.</p>
<span>Help Us Teach More Veterans</span>
<span>Learn Programming</span>
<span>Become A Mentor</span>
